src/scanpy/plotting/_stacked_violin.py

```diff
@@ -727,10 +727,6 @@ def stacked_violin(
         See :func:`~seaborn.stripplot`.
     size
         Size of the jitter points.
-    order
-        Order in which to show the categories. Note: if `dendrogram=True`
-        the categories order will be given by the dendrogram and `order`
-        will be ignored.
```

Comment:
Also,isn't this a breaking change?  This is a public API, no?

Reply:
https://scanpy.readthedocs.io/en/stable/generated/scanpy.pl.stacked_violin.html So I see that we have the `categories_order` docstring (not sure how - common args?) but `order` still seems to be a public API?  And now it's gone: https://icb-scanpy--3206.com.readthedocs.build/en/3206/generated/scanpy.pl.stacked_violin.html?

Reply:
It’s a public API … that used to be completely ignored. I can add a warning like

> `order` is ignored. use `categories_order` instead

Reply:
Interesting ok! In that case, yea a warning would be better than removal

Reply:
OK, moved to #3252